hexrd/ui/resources/calibration/default_pxrdip_config.yml

```diff
@@ -8,8 +8,8 @@ detectors:
       rows: 750
       size: [0.1, 0.1]
     transform:
-      tilt: [0.0, 0.0, 0.0]
-      translation: [0.0, 0.0, 0.0]
+      tilt: [-1.2091995761561454e+00, 1.2091995761561454e+00, -1.2091995761561454e+00]
```

Comment:
Can we round these? For instance: `[-1.21, 1.21, -1.21]`, or do they need to be very precise?

Reply:
This is 60 degree rotation around the [1,1,1] axis. Rounding to 1.21 will make it slightly larger than 60 degree rotation. So depends on how close we want it to be a 60 degree rotation.